src/.eslintrc.js

```diff
@@ -17,5 +17,49 @@
 module.exports = {
   'rules': {
     'local/no-global': 2,
+
+    'import/no-restricted-paths': [
+      'error',
+      {
+        'zones': [
+          {
+            // Disallow importing AMP dependencies into core
+            'target': 'src/core',
+            'from': 'src',
+            'except': ['./core'],
```

Comment:
Paths for `target` and `from` are absolute; paths in `except` are relative to the `from` path. Weird, I know.